apps/machine/zynqmp/platform_info.c

```diff
@@ -210,6 +221,12 @@ int platform_poll(void *priv)
 
 	prproc = rproc->priv;
 	while(1) {
+#ifdef RPMSG_NO_IPI
+		if (metal_io_read32(prproc->shm_poll_io, 0)) {
+			remoteproc_get_notification(rproc, RSC_NOTIFY_ID_ANY);
```

Comment:
Would be nice here to propagate the error returned by remoteproc_get_notification... ( in both cases)

Reply:
fixed